packages/jest-cli/src/TestRunner.js

```diff
@@ -16,24 +16,27 @@ import type {
 } from 'types/TestResult';
 import type {Config, Path} from 'types/Config';
 import type {HasteResolverContext} from 'types/Runtime';
-
-const Test = require('./Test');
+import type JestReporterBase from './reporters/JestReporterBase';
 
 const fs = require('graceful-fs');
 const getCacheFilePath = require('jest-haste-map').getCacheFilePath;
+const JestReporterCoverage = require('./reporters/JestReporterCoverage');
+const JestReporterDefault = require('./reporters/JestReporterDefault');
+const JestReporterNotify = require('./reporters/JestReporterNotify');
+const JestReporterSummary = require('./reporters/JestReporterSummary');
+const JestReporterVerbose = require('./reporters/JestReporterVerbose');
 const promisify = require('./lib/promisify');
 const snapshot = require('jest-snapshot');
+const Test = require('./Test');
 const workerFarm = require('worker-farm');
 
-type Options = {
-  maxWorkers: number,
-};
+type Options = {maxWorkers: number};
```

Comment:
this change seems unrelated. Can you revert it? If we add new options, it'll only be a one line diff instead of a 4 line diff.